provider/pktgen_utils.py

```diff
@@ -42,7 +42,7 @@ def configure_pktgen(self, params, script, pkt_cate, test_vm,
             else:
                 self.dsc = vm.get_mac_address(0)
             self.interface = vm.get_ifname(0)
-            self.runner = process.getoutput
+            self.runner = process.system_output
```

Comment:
`system_output` returns `bytes-like`, `getoutput` returns string directly, why change this?

Reply:
Since pktgen command does not stop, it is necessary to trace CmdError to the subsequent step.

Reply:
Is there any connection between "retrun type" and "trace CmdError"? Do you want `ignore_status=False`?

Reply:
Indeed, it need to set 'ignore_status=False' to trace CmdError, and 'system_output' set by default.

Reply:
OK, so we have 2 options:
1. Use system_output, but need to take care the return object, as it returns bytes-like
2. Use getoutput with `ignore_status=False`, but it already returns string type.

Reply:
> OK, so we have 2 options:
> 
> 1. Use system_output, but need to take care the return object, as it returns bytes-like
> 2. Use getoutput with `ignore_status=False`, but it already returns string type.
Perhaps it can use asystem function instead, as it doesn't handle any output, it's only requiring the tracking of CmdError for next steps. and I have tested, it works as expected.

Reply:
> 1. system_output

I suggest continuing to use the system_output since it does not require handling the output.

Here the log for using  system_output: 

[stdlog] 2024-04-15 02:52:45,714 avocado.utils.process process          L0657 INFO | Running '/tmp/pktgen_perf/pktgen_perf.sh -i t0-rPL39e -d $ip t 1 -s 64'
[stdlog] 2024-04-15 02:52:45,731 avocado.utils.process process          L0473 DEBUG| [stdout] pktgen                 73728  0
[stdlog] 2024-04-15 02:52:45,859 avocado.utils.process process          L0473 DEBUG| [stdout] Adding queue 0 of -i
[stdlog] 2024-04-15 02:52:45,993 avocado.utils.process process          L0473 DEBUG| [stdout] Configuring devices t0-rPL39e@0
[stdlog] 2024-04-15 02:52:46,010 avocado.utils.process process          L0473 DEBUG| [stdout] Running... ctrl^C to stop

Reply:
Both are up to you, I am okay for all.